ontology/1.0/ontology.owx

```diff
@@ -14,4 +14,16 @@
     <Prefix name="rdfs" IRI="http://www.w3.org/2000/01/rdf-schema#"/>
     <Import>https://ontology.cybersecuritycertcluster.eu/core</Import>
     <Import>https://ontology.cybersecuritycertcluster.eu/resource</Import>
-</Ontology>
\ No newline at end of file
+    <SubClassOf>
+        <Class IRI="/classes/PasswordBasedAuthentication"/>
```

Comment:
Please move it to the `core/security.owx` file and add the `rdfs:label` and `rdfs:comment` Annotation Assertion.